README.md

```
@@ -1,61 +1,35 @@
 
-- [Air Quality Data Guide](docs/AIR_QUALITY.md)
-- [Weather Data Guide](docs/WEATHER.md)
-- [Traffic Data Guide](docs/TRAFFIC.md)
-- [Architecture Overview](ARCHITECTURE.md)
+## Docs
 
-MIT License
\ No newline at end of file
+- [Air Quality](docs/AIR_QUALITY.md)
+- [Weather](docs/WEATHER.md)
+- [Traffic](docs/TRAFFIC.md)
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)